### PR TITLE
tickets/DM-29876: Added the ATWhiteLightSource CSC to Jenkinsfile.broker and to src/org…

### PR DESCRIPTION
**Pull request by rbovill**
…/lsst/ts/jenkins/components/Csc.groovy.

### CODE REVIEW

File: src/org/lsst/ts/jenkins/components/Csc.groovy [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+    slack_ids = ["ts-ataos":"U72CH91L2", "ts-atdome":"U2JPAP0F6", "ts-athexapod": "UAS4QHFSB", "ts-atmcs-simulator": "U2JPAP0F6", "ts-atpneumaticssimulator": "U2JPAP0F6", "ts-atspec": "U2JPAP0F6", "ts-atwhitelightsource": "cwinslow@lsst.org", "ts-cbp": "UAS4QHFSB","ts-ddsconfig": "U72CH91L2", "ts-dimm": "URY8ACN4S", "ts-dsm": "U2JPDUE86", "ts-eas": "URY8ACN4S", "ts-electrometer": "UAS4QHFSB", "ts-ess": "URY8ACN4S", "ts-externalscripts": "U72CH91L2", "ts-fiberspectrograph": "U2JPAP0F6", "ts-genericcamera": "URY8ACN4S", "ts-hexrotcomm": "U2JPAP0F6","ts-hvac": "URY8ACN4S","ts-idl": "U6BCN6H43", "ts-m2": "U72CH91L2", "ts-mtdome": "U2JPAP0F6", "ts-mtdometrajectory":"U2JPAP0F6", "ts-mteec": "URY8ACN4S", "ts-mthexapod": "U2JPAP0F6", "ts-mtmount": "U2JPAP0F6", "ts-mtrotator": "U2JPAP0F6", "ts-observatory-control": "U72CH91L2", "ts-salkafka": "U72CH91L2","ts-salobj": "U6BCN6H43", "ts-scriptqueue": "U72CH91L2", "ts-simacutuators": "U2JPAP0F6", "ts-standardscripts": "U72CH91L2","ts-dds": "U72CH91L2", "ts-tunablelaser": "UAS4QHFSB", "ts-watcher": "U2JPAP0F6", "ts-weatherstation": "URY8ACN4S"]
```
Comment (couger01) [inline src/org/lsst/ts/jenkins/components/Csc.groovy]: Need to use "U6BCN6H43" for Colin's Slack ID.

**Review by couger01 — CHANGES_REQUESTED** [review]
Only one fix otherwise looks good to me.

File: src/org/lsst/ts/jenkins/components/Csc.groovy (hunk 1 above)
Comment (rbovill) [reply src/org/lsst/ts/jenkins/components/Csc.groovy]: Fixed.

**Review by couger01 — APPROVED** [review]
LGTM.

**Pull request merged**